google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/UploadStoreSalesTransactions.java

```diff
@@ -0,0 +1,505 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.ads.googleads.examples.remarketing;
+
+import com.beust.jcommander.Parameter;
+import com.google.ads.googleads.examples.utils.ArgumentNames;
+import com.google.ads.googleads.examples.utils.CodeSampleParams;
+import com.google.ads.googleads.lib.GoogleAdsClient;
+import com.google.ads.googleads.v3.common.OfflineUserAddressInfo;
+import com.google.ads.googleads.v3.common.StoreSalesMetadata;
+import com.google.ads.googleads.v3.common.StoreSalesThirdPartyMetadata;
+import com.google.ads.googleads.v3.common.TransactionAttribute;
+import com.google.ads.googleads.v3.common.UserData;
+import com.google.ads.googleads.v3.common.UserIdentifier;
+import com.google.ads.googleads.v3.enums.OfflineUserDataJobStatusEnum.OfflineUserDataJobStatus;
+import com.google.ads.googleads.v3.enums.OfflineUserDataJobTypeEnum.OfflineUserDataJobType;
+import com.google.ads.googleads.v3.errors.GoogleAdsError;
+import com.google.ads.googleads.v3.errors.GoogleAdsException;
+import com.google.ads.googleads.v3.resources.OfflineUserDataJob;
+import com.google.ads.googleads.v3.services.AddOfflineUserDataJobOperationsResponse;
+import com.google.ads.googleads.v3.services.CreateOfflineUserDataJobResponse;
+import com.google.ads.googleads.v3.services.GoogleAdsRow;
+import com.google.ads.googleads.v3.services.GoogleAdsServiceClient;
+import com.google.ads.googleads.v3.services.OfflineUserDataJobOperation;
+import com.google.ads.googleads.v3.services.OfflineUserDataJobServiceClient;
+import com.google.ads.googleads.v3.utils.ResourceNames;
+import com.google.common.collect.ImmutableList;
+import com.google.protobuf.BoolValue;
+import com.google.protobuf.DoubleValue;
+import com.google.protobuf.Int64Value;
+import com.google.protobuf.StringValue;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+/**
+ * Uploads offline data for store sales transactions.
+ *
+ * <p>This feature is only available to whitelisted accounts. See
```

Comment:
Can we change whitelisted to allowlisted? I think right now we're attempting to change our wording a bit.

Reply:
+1

Reply:
Good point. Fixed.